plugins/modules/ipaconfig.py

```diff
@@ -524,6 +524,11 @@ def main():
 
         result = config_show(ansible_module)
 
+        sid_is_enabled = has_enable_sid and is_enable_sid(ansible_module)
+
+        if sid_is_enabled and ansible_module.params_get("enable_sid") is False:
```

Comment:
`enable_sid` variable exists some lines below. It would be good o move it there to the other test.

Reply:
fixed